src/main/resources/i18n/messages.properties

```diff
@@ -1961,40 +1965,44 @@ ViewAnnouncement.read=Read
-metadata.set-sample-id.title=Select column header corresponding to the 'Sample Identifier'.
-
-metadata.upload.component.title=Sample Metadata Uploader
-metadata.upload.errors.repeating-headers=The file uploaded contains a duplicate header
-metadata.upload.errors.title=Error uploading Excel Spreadsheet
-metadata.upload.component.intro=The Metadata Uploader is a tool you can use to upload an excel file containing metadata for samples in this project.
-metadata.upload.component.text=To ensure that the uploaded file can be mapped correctly to samples in this project, one of the columns must contains the <b>name of the sample</b> that the data belongs to. You will be able to select that column in the next step.
-metadata.upload.component.warning=Metadata uploaded will overwrite duplicate metadata on the sample.
-metadata.upload.component.error.bad-file.title=There was an error processing the uploaded file.
-metadata.upload.component.error.bad-file.body=Please check the uploaded file to ensure that it is an Excel formatted file, either '.xls' or '.xlsx',
-
-metadata.header.description=Select which spreadsheet column maps to the sample name in IRIDA
-metadata.header.component.prev-nav=Select new Excel File
-metadata.header.component.next-nav=Preview Metadata to be Uploaded
-
-metadata.results.component.prev-nav=Select different column as Sample Name
-metadata.results.component.found-nav=Rows matching samples
-metadata.results.component.missing-nav=Rows not matching samples
-
-metadata.results.found.component.title=These rows correspond to samples within this project
-metadata.results.found.component.save=Save valid metadata
-metadata.results.found.component.save-btn-active=Saving ...
-metadata.results.found.component.empty-title=None of the rows match samples in this project.
-metadata.results.found.component.empty-text=Check the column you selected as corresponding to the sample name, and make sure that you are uploading to the correct project.
-
-metadata.results.missing.component.title=These rows do not correspond to samples within this project
-metadata.results.missing.component.message=This data will not be saved.  You could check the sample name column selected to ensure that the values are correct.
-metadata.results.missing.none-title=Success!
-metadata.results.missing.none-text=All rows mapped to samples in this project, nothing to worry about.
-
-metadata.results.save.found-error=Since none of the metadata could match existing samples, nothing could be save.
-metadata.results.save.sample-not-found=Sample could not be found: {0}
-metadata.results.save.success={0} samples have had metadata added to them.
+
+server.metadataimport.results.save.success.single-updated={0} sample has been updated
+server.metadataimport.results.save.success.multiple-updated={0} samples were updated
+server.metadataimport.results.save.success.single-created=\ and {0} new sample was created.
```

Comment:
I did not notice this before, but this was a really smart way to handle this!

Reply:
Thanks.